docs/examples/update-vectors.rst

```diff
@@ -154,28 +154,30 @@ C
         c_int n = 2;
         c_int m = 3;
 
-        // Problem settings
-        OSQPSettings * settings = (OSQPSettings *)c_malloc(sizeof(OSQPSettings));
+        // Exitflag
+        c_int exitflag = 0;
```

Comment:
I know this is only documentation, but assigning variables before declaring all the variables is not compatible with ANSI C and I would avoid it also in the actual C code.

Reply:
It should be allowed to assign values to variables when you declare them:
https://stackoverflow.com/questions/32786702/c89-and-variable-initialization

Compiling the code above with `gcc` with `-ansi` and `-pedantic` options generates no warnings.

Reply:
Ok that's good then. I remember getting such errors with similar code before but I guess it was related to assignments without declarations.